ptf/saiswitch.py

```diff
@@ -83,38 +83,17 @@ def generate_mac_list(no_of_addr):
                         return mac_list
     return mac_list
 
-
-@group("draft")
-class SwitchAttrTest(SaiHelper):
-    '''
-    Switch attributes tests
-    '''
-
+class AvailableIPv4RouteEntryTest(PlatformSaiHelper):
     def setUp(self):
-        super(SwitchAttrTest, self).setUp()
+        super(AvailableIPv4RouteEntryTest, self).setUp()
 
         # values required by neighbor entries tests set by route entries test
         # Note: routes entries tests should be run as first
         self.available_v4_host_routes = None
         self.available_v6_host_routes = None
 
-    def runTest(self):
-        self.availableIPv4RouteEntryTest()
-        self.availableIPv6RouteEntryTest()
-        self.availableIPv4NexthopEntryTest()
-        self.availableIPv6NexthopEntryTest()
-        self.availableIPv4NeighborEntryTest()
-        self.availableIPv6NeighborEntryTest()
-        self.availableNexthopGroupEntryTest()
-        self.availableNexthopGroupMemberEntryTest()
-        self.availableFdbEntryTest()
-        self.availableAclTableTest()
-        self.readOnlyAttributesTest()
-        self.refreshIntervalTest()
-        self.availableSnatEntryTest()
-        self.availableDnatEntryTest()
 
-    def availableIPv4RouteEntryTest(self):
+    def check_data_creation(self):
```

Comment:
If this is a subtests method the name should be in camelCase style. 
same for check_amount